src/navigator/DrawerNavigator.tsx

```diff
@@ -304,7 +304,7 @@ export default function DrawerNavigator({ route }: Props) {
               <View style={styles.itemStyle}>
                 <Text style={styles.itemTitle}>{t('walletSecurity')}</Text>
                 <View style={styles.drawerItemIcon}>
-                  <ExclamationCircleIcon />
+                  <AttentionIcon color={Colors.primary} />
```

Comment:
Looks like we should pass `size={20}` since it was the size used in `ExclamationCircleIcon`.

Reply:
fixed, thanks!